src/deploy/RPM_build/noobaa.spec

```diff
@@ -101,6 +101,11 @@ if [ "${state}" == "active" ]; then
   service rsyslog restart
 fi
 
+while true; do
+    /usr/sbin/logrotate /etc/logrotate.d/noobaa 2>&1
+    sleep 30
+done &
+
```

Comment:
This is not a good place to set up long running tasks, it will not persist after reboot or failures.

The recommended way to change logrotate schedule is to change it's system wide cron schedule which is daily by default using something like - 
```
sudo mv /etc/cron.daily/logrotate /etc/cron.hourly
```
(see https://stackoverflow.com/questions/25485047/hourly-rotation-of-files-using-logrotate and https://betterstack.com/community/guides/logging/how-to-manage-log-files-with-logrotate-on-ubuntu-20-04/#changing-the-system-logrotate-schedule)

But this is not something that makes sense to add to our RPM and should be handled by the system admin.

For our use case, what we want is to trigger rotation by size, which could be done using rsyslog outchannel trigger that will invoke logrotate - see reference https://www.rsyslog.com/doc/tutorials/log_rotation_fix_size.html which would be added to rsyslog config like:
```
$outchannel noobaa_log,/var/log/noobaa.log, 52428800,/path/to/logrotate_noobaa
```
where logrotate_noobaa script calls `/usr/sbin/logrotate /etc/logrotate.d/noobaa` ...

Reply:
code updated